### PR TITLE
Add info on setting up activity player support

### PR DESCRIPTION
**Pull request by scytacki**

### CODE REVIEW

**Comment by scytacki** [comment]
@pjanik I don't think most of these changes would help with the issue you were seeing yesterday. The only one that might is making sure that `REPORT_SERVICE_URL` is set in LARA.

File: README.md [hunk 1]
```diff
@@ -103,15 +105,38 @@ With the following settings you can:
    3. Update the external report called DEFAULT_REPORT_SERVICE. It should have a URL of: https://portal-report.concord.org/branch/master/index.html?sourceKey=app.lara.docker.username **Replace `username` with the username on your local system. If you don't know your username, run `echo $USER`**.
 4. in LARA: `cp .env-osx-sample .env`
 5. in LARA `.env`:
-    1. set `REPORT_SERVICE_TOKEN` (see the comment in the .env file)
+    1. set `REPORT_SERVICE_URL` (see the value in the .env-osx-sample file)
+    1. set `REPORT_SERVICE_TOKEN` (see the comment in the .env-osx-sample file)
     2. set `LARA_PROTOCOL=https`
     3. set `PORTAL_PROTOCOL=https`
```
Comment (kswenson) [inline README.md:111]: You probably want to number these 1-4 (or make them all 1s so they auto-increment).

Comment (scytacki) [reply README.md]: Thanks @kswenson. I realized I actually added this to the wrong section.

**Review by pjanik — APPROVED** [review]
Looks good, it'd address my issues, thanks. :)

**Review by tejal-shah — APPROVED** [review]
Looks good to me, thanks!

**Pull request merged**